app/views/contexts/summary/question.py

```diff
@@ -39,7 +39,16 @@ def __init__(
         )
 
     def _get_answer(self, answer_store, answer_id):
-        return answer_store.get_escaped_answer_value(answer_id, self.list_item_id)
+        if answer_store.get_escaped_answer_value(
+            answer_id, self.list_item_id
+        ) or isinstance(
+            answer_store.get_escaped_answer_value(answer_id, self.list_item_id),
+            int,  # protection against falsy answer value "0"
+        ):
+            return answer_store.get_escaped_answer_value(answer_id, self.list_item_id)
+        if self.schema.get_default_answer(answer_id):
+            return self.schema.get_default_answer(answer_id).value
+        return None
```

Comment:
What is coming back from the default answer? As you should be just be able to just return self.schema.get_default_answer(answer_id) and forget the rest

Reply:
It returns an Answer class instance with `answer_id`, `list_item_id` and `value` so we only need `value` key here.

Reply:
Ok, but you could also tidy this up a little, something like 
```    
def _get_answer(self, answer_store, answer_id):
        escaped_answer_value = answer_store.get_escaped_answer_value(answer_id, self.list_item_id)

        if escaped_answer_value or isinstance(escaped_answer_value, int):
            return escaped_answer_value

        default_answer = self.schema.get_default_answer(answer_id)
        return default_answer.value if default_answer else None
```

Reply:
Do you even need instance, isn't it just for 0?